src/directives/Tooltip/index.js

```diff
@@ -20,15 +20,12 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+import { VTooltip, options } from 'floating-vue'
```

Comment:
So in talk we were setting the default container:
```js
Tooltip.options.defaultContainer = '#content-vue'
```

But with floating-vue the options are not on the exported object anymore.
So can we also export the options Object?

Reply:
background for defaultContainer: to make sure the tooltips stay visible even in fullscreen mode